chainerx_cc/chainerx/routines/manipulation.cc

```diff
@@ -737,6 +737,11 @@ Array ExpandDims(const Array& a, int8_t axis) {
     return out;
 }
 
+Array Flatten(const Array& a) {
+    Array out = a.Reshape({a.GetTotalSize()});
```

Comment:
`numpy.ndarray.flatten` always return their copy.
```suggestion
    Array out = a.Copy().Reshape({a.GetTotalSize()});
```

Reply:
Fixed.

Reply:
Why didn't you apply this fix? The code in my suggestion seems simpler.

Reply:
Fixed now. Please see if it's alright now.